### PR TITLE
feat: add support for Gaudi 2 Accelerator

### PR DESCRIPTION
**Pull request by kyujin-cho**
### TL;DR

This PR adds support for dealing with Gaudi 2 accelerator from WebUI.

### What changed?

- Parse accelerators in agent list page by checking only `available_slots`
- Added 'Gaudi 2' to the spell checker configuration.
- Updated `config.toml.sample` to include Gaudi 2 device limit configurations.
- Modified several TypeScript components (`ResourceNumber.tsx`, `ServiceLauncherPageContent.tsx`, `backendai.tsx`, and others) to handle Gaudi 2 devices.
- Updated the device metadata JSON to describe Gaudi 2 devices.
- Added multilingual support entries for Gaudi 2 devices in multiple languages.
- Removed outdated components (`backend-ai-agent-list.ts`, `backend-ai-agent-view.ts`) no longer in use.
- Made adjustments in various lists and settings to account for the addition of Gaudi 2 devices.

### How to test?
1. If you want to know the test setting, please ask @kyujin-cho 
2. Check if a new session has been successfully created.
3. Check the session list, agent summary, agent list, environment image list, configuration page and resource panel supports Gaudi 2.

### Why make this change?

The addition of Gaudi 2 device support extends the system's capabilities, allowing it to handle new hardware configurations and improve overall performance.

### Screenshots
- session launcher
image
image

- session list
image

- agent summary
image

- agent list
image

- environment - images
image

- resource panel
image

- configurations
image

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [x] Specific setting for review (eg., KB link, endpoint or how to setup)
- [x] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
* **#2623** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2623?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kyujin-cho and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/helper/index.tsx [hunk 1]
```diff
@@ -171,7 +171,7 @@ export function iSizeToSize(
   const sizes = ['B', 'K', 'M', 'G', 'T', 'P', 'E'];
   const [sizeValue, sizeUnit] = parseUnit(sizeWithUnit);
   const sizeIndex = sizes.indexOf(sizeUnit.toUpperCase());
-  if (sizeIndex === -1 || isNaN(sizeValue)) {
+  if (sizeIndex === -1 || isNaN(sizeValue) || !sizeValue) {
```
Comment (yomybaby) [inline react/src/helper/index.tsx]: @agatha197, Can I ask why did you add the `!sizeValue` condition? This causes an error when `iSizeToSize("0g","b");` is called.

**Review by yomybaby — CHANGES_REQUESTED** [review]

**Review by yomybaby — COMMENTED** [review]
After this PR, the Resource Number is displayed with the `~∞` even where it is not needed.
image

**Review by ironAiken2 — CHANGES_REQUESTED** [review]
If you run a session with accelerator set to 1 or higher, the session list does not reflect the accelerator.

**Comment by agatha197** [comment]
> If you run a session with accelerator set to 1 or higher, the session list does not reflect the accelerator.
> 
> 

It is core issue. It will be handled in https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2682/fix-Remove-unused-device-gaudi-device.

**Review by ironAiken2 — APPROVED** [review]
LGTM

**Pull request merged**